modPrecompiled/src/org/aion/precompiled/contracts/ContractBase.java

```diff
@@ -0,0 +1,5 @@
+package org.aion.precompiled.contracts;
+
+public class ContractBase {
```

Comment:
is this file needed?

Reply:
cause I put a static method originally from to bridge contract to here, there are some contracts can use the same statics method but in the different package.